cfme/configure/access_control.py

```diff
@@ -576,9 +576,9 @@ def create(self, cancel=False):
 
         navigate_to(self, 'Add')
         fill(self.tenant_form, self, action=form_buttons.add)
-        if type(self) is Tenant:
+        if isinstance(self, Tenant):
```

Comment:
We should check those carefully as there might be something that wants to react only on that specific class and not subclasses.